code/modules/clothing/glasses/glasses.dm

```diff
@@ -314,3 +314,18 @@
 	icon_state = "kaminaglasses"
 	item_state = "kaminaglasses"
 	cover_hair = 1
+
+/obj/item/clothing/glasses/contacts
+	name = "Contact Lenses"
+	desc = "Only nerds wear glasses."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "contact"
+	prescription = 1
```

Comment:
use TRUE

Reply:
all other forms of glasses use 1 instead of true